.buildkite/bk.integration.pipeline.yml

```diff
@@ -14,6 +14,31 @@ env:
   IMAGE_WIN_2022: "platform-ingest-elastic-agent-windows-2022-1749258065"
   IMAGE_WIN_2025: "platform-ingest-elastic-agent-windows-2025-1749258065"
 
+# This section is used to define the plugins that will be used in the pipeline.
+# See https://buildkite.com/docs/pipelines/integrations/plugins/using#using-yaml-anchors-with-plugins
+common:
+  - google_oidc_plugin: &google_oidc_plugin
+      # See https://github.com/elastic/oblt-infra/blob/main/conf/resources/repos/elastic-agent/01-gcp-oidc.tf
+      # This plugin authenticates to Google Cloud using the OIDC token.
+      elastic/oblt-google-auth#v1.3.0:
+        lifetime: 10800 # seconds
+        project-id: "elastic-observability-ci"
+        project-number: "911195782929"
+# see https://github.com/avaly/gcp-secret-manager-buildkite-plugin/pull/10
+# see https://github.com/avaly/gcp-secret-manager-buildkite-plugin/pull/11
+#  - gcp_serverless_secrets_plugin: &gcp_serverless_secrets_plugin
```

Comment:
Q: As I understand, you've raised a PR, and we temporarily use a forked plugin, right?